app/services/rollover_reporting_service.rb

```diff
@@ -0,0 +1,72 @@
+class RolloverReportingService
+  def initialize
+    @rollover_scope = RecruitmentCycle.current_recruitment_cycle
+  end
+
+  class << self
+    def call
+      new.call
+    end
+  end
+
+  def call
+    if @rollover_scope.next.present?
+      {
+      total: {
+        published_courses: published_courses_count,
+        new_courses_published: new_courses_published_count,
+        deleted_courses: deleted_courses_count,
+        existing_courses_in_draft: existing_courses_in_draft_count,
+        existing_courses_in_review: existing_courses_in_review,
+      },
+    }
+    else
+      {}
+    end
```

Comment:
this don't look like its tests....

also is there an actual need to return anything?

Reply:
fixed, cheers